docs/_pages/docs/03-understand/07-wallets.md

```diff
@@ -76,6 +77,10 @@ Experience using Electrum is limited. It does not seem to support more than one
 ### LND (CLI Interface)
 Raw; it shows exactly what is happening and what it knows "IN_FLIGHT". It is not user friendly and therefore not recommended to interact with RoboSats by beginners. However, everything works just fine. If you are using LNCLI regularly, then you will find no issue using it with RoboSats.
 
+
+### Mash Wallet App (Mobile PWA & Desktop Web-Wallet)
+Overall the [Mash](https://mash.com/consumer-experience/) wallet works end2end with Robosats on both selling & receiging with all details in the notes for the invoices showin. The details of the flows in the app are shown. When the transactions are complete, they open in the mobile app on both sender/receiver sides to highlight that the transactions are completed.The one UX hick-up is that the pending invoices list doesn't explicitly show HOLD invoices (they are filtered), but will open a bug to fix this issue shortly (this note is from Aug 21st 2023). 
```

Comment:
Only a couple of typos, `receiging`, `showin` and a bit of dislexia `open a bug to fix this issue` :smile:

Reply:
Fixed typos.